src/components/screens/transactionDetails/layoutSchema.js

```diff
@@ -12,48 +12,50 @@ const {
   // reclaimLSK,
 } = MODULE_ASSETS_NAME_ID_MAP;
 
-const baseComponents = [Sender, Confirmations, TransactionId, Fee, Date, BlockId, BlockHeight];
-const previewBaseComponents = [Sender, TransactionId, Fee, SignedAndRemainingMembersList];
+const baseComponents = [Illustration, Sender];
+const timeComponents = [TransactionId, Date, BlockId, BlockHeight, Fee, Confirmations];
+const previewBaseComponents = [Illustration, Sender];
+const restComponents = [TransactionId, Fee, SignedAndRemainingMembersList];
 
 const LayoutSchema = {
   [transfer]: {
-    components: [...baseComponents, Recipient, Illustration, Amount, Message],
-    className: '',
+    components: [...baseComponents, Recipient, Amount, Message, ...timeComponents],
```

Comment:
The order in which the layout is shown does not depend on the order of components here. If this is just for cleanup, then its fine

Reply:
Mostly clean up. It helped to have a more unified detail view